scripts/mitochondria_m2_wdl/AlignAndCall.wdl

```diff
@@ -469,7 +469,7 @@ task M2 {
         --max-mnp-distance 0
   >>>
   runtime {
-      docker: select_first([gatk_docker_override, "us.gcr.io/broad-gatk/gatk:4.1.7.0"])
```

Comment:
@ahaessly This is ok to upgrade I'm assuming?

Reply:
yes